Formula/cql.rb

```diff
@@ -0,0 +1,23 @@
+class Cql < Formula
+  desc "Decentralized SQL database with blockchain features"
+  homepage "https://covenantsql.io"
+  url "https://github.com/CovenantSQL/CovenantSQL/archive/v0.5.0.tar.gz"
+  sha256 "ebee3e8fca672d6f3d3d607e380ec7a1f3a31501e5c80b85c660f50eda7a2240"
+  head "https://github.com/CovenantSQL/CovenantSQL.git"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["CQLVERSION"] = "v0.5.0"
+    mkdir_p "src/github.com/CovenantSQL"
+    ln_s buildpath, "src/github.com/CovenantSQL/CovenantSQL"
+    system "make", "bin/cql"
+    bin.install "bin/cql"
+  end
+
+  test do
+    system bin/"cql", "help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the [formula cookbook](https://docs.brew.sh/Formula-Cookbook).

In most cases, a good test would involve running a simple test case: run `#{bin}/foo input.txt`.
- Then you can check that the output is as expected (with `assert_equal` or `assert_match` on the output of `shell_output`)
- You can also check that an output file was created, if that is expected: `assert_predicate testpath/"output.txt", :exist?`

Some advice for specific cases:
- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Fixed